index.html

```diff
@@ -56,25 +49,37 @@
             </form>
           </div>
         </div>
-          <div id="app"></div>
       </section>
-      <section id="tab2" class="tab-content hide">
-          <div class="aboutMe">
-              <p>This is my story.</p>
-          </div>
+
+      <section id="about" class="tab-content">
+        <div id="app"></div>
+        <div class="aboutMe">
+            <p>This is my story.</p>
+        </div>
       </section>
-      <section id="tab3" class="tab-content hide">
-        <section id="stats"><p>This is a test</p></section>
+
+      <section id="stats" class="tab-content">
+        <p>This is a test</p>
       </section>
+
     </main>
     <footer>
+      <div>
+        <!-- &copy; 2015 Code Fellows | Static Blog | Happy Ipsum! -->
+      </div>
     </footer>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.js" charset="utf-8"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.js" charset="utf-8"></script>
+    <script src="https://cdn.rawgit.com/visionmedia/page.js/master/page.js"></script>
     <script src="vendor/scripts/html5sql.js" type="text/javascript"></script>
-    <script src="vendor/scripts/webDB.js" type="text/javascript"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/marked/0.3.5/marked.min.js"></script>
```

Comment:
You may find it useful to organize this section of scripts as Model, View, Controller, and Router, as we demoed in class.